examples/main-lualatex.tex

```diff
@@ -30,10 +30,11 @@
 \gresetlinecolor{gregoriocolor}
 
 % We set VII above the initial.
-\greannotation{\small \textsc{\textbf{VII}}}
+\grechangestyle{annotation}{\small\scshape\bfseries}
```

Comment:
We could, but then the example doesn't demonstrate the use of `\greannotation` (something the Factus example below already demonstrates).  Perhaps what is needed is a slightly more complicated annotation which can't simply be imported from the mode field.  Maybe a two line annotation like:
```
\grechangestyle{annotation}{\small\bfseries}
\greannotation{Intr.}
\greannotation{\textsc{VII}}
```